internal/post/message.go

```diff
@@ -70,14 +75,21 @@ func (msg *Message) Bytes() []byte {
 		var sub io.Writer
 
 		if len(msg.PlainText) > 0 {
-			sub, _ = wr.CreatePart(textproto.MIMEHeader{
+			sub, err := wr.CreatePart(textproto.MIMEHeader{
 				"Content-Type":              []string{"text/plain; charset=UTF-8; format=flowed"},
 				"Content-Transfer-Encoding": []string{"quoted-printable"},
 			})
+			if err != nil {
+				return nil, Error.Wrap(err)
+			}
 
 			enc := quotedprintable.NewWriter(sub)
-			_, _ = enc.Write([]byte(msg.PlainText))
-			_ = enc.Close()
+			_, err = enc.Write([]byte(msg.PlainText))
+			if err != nil {
+				return nil, Error.Wrap(err)
+			}
+
+			defer func() { err = errs.Combine(err, enc.Close()) }()
```

Comment:
This defer should be just after the `enc` initialization for guaranteeing that always gets closed.